.goreleaser.yml

```diff
@@ -308,4 +346,27 @@ brews:
     homepage: https://docs.confluent.io/confluent-cli/current/overview.html
     description: CLI for Confluent Cloud and Confluent Platform
     skip_upload: "{{ .Env.DRY_RUN }}"
-    test: system "#{bin}/confluent version"
\ No newline at end of file
+    test: system "#{bin}/confluent version"
+
+chocolateys:
+  # TODO: Check naming requirements for Chocolatey
+  - name: confluent
+    ids:
+      - archive-chocolatey
+    owners: confluentinc
+    title: Confluent CLI
+    authors: Confluent Inc.
+    project_url: https://docs.confluent.io/confluent-cli/current/overview.html
+    url_template: "https://s3-us-west-2.amazonaws.com/confluent.cloud/confluent-cli/archives/{{ .Version }}/{{ .ArtifactName }}"
+    copyright: 2023 Confluent Inc.
+    license_url: https://github.com/confluentinc/cli/blob/main/LICENSE
+    require_license_acceptance: false
+    project_source_url: https://github.com/confluentinc/cli
+    bug_tracker_url: https://github.com/confluentinc/cli/issues
+    summary: CLI for Confluent Cloud and Confluent Platform
+    description: CLI for Confluent Cloud and Confluent Platform
+    release_notes: https://docs.confluent.io/confluent-cli/current/release-notes.html
+    # TODO
+    api_key: "{{ .Env.CHOCOLATEY_API_KEY }}"
```

Comment:
Should we be reading this directly from `vault` instead?

Reply:
Yeah if we do end up using the chocolatey repo, that's what we'll do. This is just a "don't forget this" placeholder.